### DISCUSSION TITLE
Which CoinType does opbnb correspond to?

### DISCUSSION THREAD

**Discussion by adam-alex-albert** [trustwallet/wallet-core#3768] [Q&A]

**Comment by satoshiotomakan** [comment]
Hi @adam-alex-albert, it corresponds to https://opbnb.bnbchain.org/en

**Reply by adam-alex-albert** [reply]
<img width="895" alt="image" src="https://github.com/trustwallet/wallet-core/assets/147524989/8ac18bdd-eba1-46ea-b5dc-e988627d3110">
Which one does it correspond to in CoinType?

**Reply by satoshiotomakan** [reply]
`CoinType.opBNB`

**Reply by adam-alex-albert** [reply]
<img width="865" alt="image" src="https://github.com/trustwallet/wallet-core/assets/147524989/d6523039-8ebb-4968-bca1-c915bab34639">
Swift version 4.0.32

**Reply by satoshiotomakan** [reply]
`.opBNB` should work on Swift and Kotlin:
https://github.com/trustwallet/wallet-core/blob/adb20c0577bc8c5dce5e6bb6d865c3727a4d9e96/swift/Tests/CoinAddressDerivationTests.swift#L108

**Reply by satoshiotomakan** [reply]
Do you compile WalletCore locally, or use an SPM/Cocoapods package?
If compile it locally, please make sure you ran `./tools/generate-files` to generate Swift bindings

**Reply by adam-alex-albert** [reply]
<img width="927" alt="image" src="https://github.com/trustwallet/wallet-core/assets/147524989/f94c74e7-8cd2-49be-b218-68ab2b3aae24">
spm,   Project in xcode

**Reply by adam-alex-albert** [reply]
Import in Cocoapods mode, and everything is normal.
